docs/quick-start/configure-mongodb.txt

```diff
@@ -0,0 +1,56 @@
+.. _laravel-quick-start-connect-to-mongodb:
+
+=================================
+Configure Your MongoDB Connection
+=================================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: test connection, runnable, code example
+
+.. procedure::
+   :style: connected
+
+   .. step:: Set the Connection String in the Database Configuration
+
+      Open the ``database.php`` file in the ``config`` directory and
+      set the default database connection to ``mongodb`` as shown
+      in the following line:
+
+      .. code-block:: php
+
+         'default' => env('DB_CONNECTION', 'mongodb'),
+
+      Add the following highlighted ``mongodb`` entry to the ``connections`` array
+      in the same file. Replace the ``<connection string>`` placeholder with the
+      connection string that you copied from the :ref:`laravel-quick-start-connection-string`
+      step in the following code example:
+
+      .. code-block:: php
+         :emphasize-lines: 2-6
+
+         'connections' => [
+           'mongodb' => [
+             'driver' => 'mongodb',
+             'dsn' => env('DB_URI', '<connection string>'),
+             'database' => 'sample_mflix',
+           ],
+
+         // ...
+
+      .. step:: Add the Laravel MongoDB Provider
```

Comment:
I: I think there might be an indentation issue with this step, since I don't see it on the staging page

Reply:
Good catch! Strange that this didn't report an issue in the autobuilder log.